include/git2/sys/refdb_backend.h

```diff
@@ -107,7 +107,11 @@ struct git_refdb_backend {
 	 * from the refdb.  A refdb implementation must provide this
 	 * function.
 	 */
-	int (*del)(git_refdb_backend *backend, const char *ref_name, const git_oid *old_id, const char *old_target);
+	int (*del)(
+		git_refdb_backend *backend,
+		const char *ref_name,
+		const git_signature *who, const char *message,
+		const git_oid *old_id, const char *old_target);
```

Comment:
This is unfortunate, as the change breaks the API. I'd rather opt for a new method, as libgit2 should finally grow up and be more serious about API stability.